std_msgs/String name

---

nav_msgs/Odometry RESULT